loops.js

```diff
@@ -51,3 +65,13 @@
 // >> 9
 // ==========================================
 
+sign = "";
```

Comment:
Ook hier ontbreekt `let`.

En ook hier is het niet nodig om een aparte variabele te declareren en aan te passen. Net als bij vorige opdracht direct loggen.

Reply:
Zonder extra variabele, en verdeeld  over 3 if's zou het zo worden:

```
for (let i = 0; i < 10; i++) {
  if (i < 3) {
    console.log(i);
  }
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }
  if (i >= 6) {
    console.log('>> ' + i);
  }
}
```